packages/fetch-and-compile/test/fetch.test.ts

```diff
@@ -0,0 +1,118 @@
+import assert from "assert";
+import "mocha";
+import Config from "@truffle/config";
+import { fetchAndCompile } from "../lib/index";
+import axios from "axios";
+import sinon from "sinon";
+const fixture: any = require("./fixture.js");
+
+function stubAxiosGetMethod(url: string, address: string, data: object) {
+  sinon
+    .stub(axios, "get")
+    .withArgs(url, {
+      params: {
+        module: "contract",
+        action: "getsourcecode",
+        address: address,
+        apikey: ""
+      },
+      responseType: "json",
+      maxRedirects: 50
+    })
+    .returns(Promise.resolve({ data: data }));
+}
+
+afterEach(function () {
+  //TS can't detect that is a sinon stub so we have to use ts-ignore
+  //@ts-ignore
+  //restoring stub
+  axios.get.restore();
+});
+
+describe("fetchAndCompile", function () {
+  it("verifes contract from mainnet", async function () {
```

Comment:
Nitpicking, but just noticed a spelling error here; "verifes" should be "verifies".